go/trivial.go

```diff
@@ -25,6 +27,10 @@ func main() {
 		"your_service_name",
 		config.Logger(jaeger.StdLogger),
 	)
+	if (err != nil){
+		fmt.Println("Cannot create tracer: " +err.Error())
```

Comment:
I think that replacing the string concatenation with `fmt.Printf("Cannot create tracer: %v\n", err)` looks a bit better

Reply:
Thanks, that's fixed now.